lib/testmap.py

```diff
@@ -139,12 +139,16 @@
         'main': [
             'fedora-33',
             'fedora-33/firefox',
+            'fedora-34',
             'rhel-8-5',
             'rhel-9-0'
         ],
+        'rhel-8-5': [
+            'rhel-8-5',
+            'rhel-8-5/firefox',
```

Comment:
This did not run (unknown context), but indeed the chance is very high that it works, too. If not, we'll have to pay the price on the next image refresh :grin: